Modules/Core/Mesh/include/itkBinaryMask3DMeshSource.hxx

```diff
@@ -2694,15 +2694,118 @@ BinaryMask3DMeshSource<TInputImage, TOutputMesh>::PrintSelf(std::ostream & os, I
 {
   Superclass::PrintSelf(os, indent);
 
-  os << indent << "ObjectValue: " << static_cast<NumericTraits<unsigned char>::PrintType>(m_ObjectValue) << std::endl;
+  os << indent << "RegionOfInterestProvidedByUser: " << (m_RegionOfInterestProvidedByUser ? "On" : "Off") << std::endl;
+  os << indent << "RegionOfInterest: " << m_RegionOfInterest << std::endl;
 
-  os << indent << "NumberOfNodes: " << m_NumberOfNodes << std::endl;
+  os << indent << "LUT: " << m_LUT << std::endl;
 
-  os << indent << "NumberOfCells: " << m_NumberOfCells << std::endl;
+  os << indent << "LastVoxel: " << static_cast<typename NumericTraits<IdentifierType>::PrintType>(*m_LastVoxel)
+     << std::endl;
+  os << indent << "CurrentVoxel: " << static_cast<typename NumericTraits<IdentifierType>::PrintType>(*m_CurrentVoxel)
+     << std::endl;
 
-  os << indent << "RegionOfInterestProvidedByUser: " << m_RegionOfInterestProvidedByUser << std::endl;
+  os << indent << "LastRow: ";
+  if (m_LastRow != nullptr)
```

Comment:
Wasn't there a `PrintNullable` macro or something like that? Repeats more times below.